WD-Firmware/comrv/loc_inc/comrv.h

```diff
@@ -35,6 +35,10 @@
 /**
 * macros
 */
+#define M_COMRV_ENTER_CRITICAL_SECTION()
+#define M_COMRV_EXIT_CRITICAL_SECTION()
+// TODO: use our psp
+#define _BUILTIN_EXPECT(condition, expected)  __builtin_expect(condition, expected)
```

Comment:
please add a comment why we are doing this

Reply:
done in b4d5e7b